xwiki-rendering-api/src/main/java/org/xwiki/rendering/internal/listener/descriptor/DefaultListenerDescriptorManager.java

```diff
@@ -115,7 +116,7 @@ public ListenerDescriptor createDescriptor(Class<?> type)
      */
     private void addElement(String elementName, ListenerDescriptor descriptor, Method method)
     {
-        String lowerElementName = elementName.toLowerCase();
+        String lowerElementName = elementName.toLowerCase(Locale.ENGLISH);
```

Comment:
Shouldn't we use Locale.ROOT instead?

See https://garygregory.wordpress.com/2015/11/03/java-lowercase-conversion-turkey/

Reply:
That makes Locale.ROOT a lot less neutral than I tough if it's dealing with latin letters uppercase/lowercase :) I find it a bit counter intuitive.

Reply:
Anyway if we are sure this is officially supported and Local.ROOT won't suddenly stop doing that the I gree it make more sense for use cases like these (since it's not exactly english content but method names).